easybuild/easyconfigs/c/chromVARmotifs/chromVARmotifs-0.2.0-foss-2023a-R-4.3.2.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'RPackage'
+
+name = 'chromVARmotifs'
+local_commit = '38bed55'
+# see DESCRIPTION file
+version = '0.2.0'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://github.com/GreenleafLab/chromVARmotifs'
+description = """The goal of chromVARmotifs is to make it easy to use several different motif collections in R,
+ particularly for use with motifmatchr and chromVAR packages."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('R', '4.3.2'),
+    ('R-bundle-CRAN', '2023.12'),
+    ('R-bundle-Bioconductor', '3.18', versionsuffix),
+]
+
+source_urls = ['https://github.com/GreenleafLab/chromVARmotifs/archive']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['6bc0894e1ffc9aa51d7eefce71ca0a586b271e7c6fd60211b179862eb51e4f1b']
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': [name],
+}
+
+modextrapaths = {'R_LIBS_SITE': ''}
```

Comment:
@boegel I'm seeing the following when building this:
```
WARNING: Suppressed adding the following path(s) to $R_LIBS_SITE of the module as they were already added: ['']
```
So I guess this line can be removed?
```suggestion
```

Reply:
Yes, this is done by `RPackage` easyblock by default, indeed.